Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,6 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+
```

Comment:
Can you also explain the meaning of `maximum - minimum + 1`? Why does it needs +1 to the difference?

If you were to write an expression or statement to generate a random number between -50 and 50 (including both numbers), how would you write the code?

Reply:
dding 1 to this difference ensures that the maximum value is included. In the previous example, you would have:

From 1 to 100: Total Count = (100 - 1) + 1 = 100.
Therefore, the total number of integers from 1 to 100 (inclusive) is 100.

Reply:
const minimum = -50;
const maximum = 50;

const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
const minimum = -50;
const maximum = 50;

const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;